src/main/java/exceptions/InvalidCostException.java

```diff
@@ -0,0 +1,13 @@
+package exceptions;
+
+/**
+ * Represents the exception thrown when the cost inputted by the user is invalid.
+ */
+public class InvalidCostException extends GitException{
```

Comment:
The exception class InvalidCostException has been added properly. It extends GitException, which fits the design of having a hierarchy of exceptions in the application. The message "Please input a valid cost!" is clear and provides straightforward feedback to the user.